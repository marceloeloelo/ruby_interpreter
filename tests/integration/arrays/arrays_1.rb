a = []

b = [1, "hola", nil]

puts (b[0])

puts (b[1])

puts (b[2])

puts (b[3])
