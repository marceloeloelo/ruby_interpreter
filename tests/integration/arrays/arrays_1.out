1
hola
nil
nil
